public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuTrigger.tsx

```diff
@@ -20,8 +20,6 @@ export const PanelHeaderMenuTrigger: FC<Props> = ({ children, ...divProps }) =>
         return;
       }
 
-      event.stopPropagation();
```

Comment:
Allowing the event to continue propagation seems to be the first step towards solving this.